numba/tests/test_ufuncs.py

```diff
@@ -1613,6 +1614,8 @@ class TestLoopTypesDatetimeNoPython(_LoopTypesTester):
     _compile_flags = no_pyobj_flags
     _ufuncs = supported_ufuncs[:]
 
+    _ufuncs.remove(np.divmod)  # not implemented yet
```

Comment:
I didn't fancy dealing with this, the main purpose of the patch was to make it possible to write multi-output ufuncs, not to fully implement all of them.